bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/RootClassifierWizard.java

```diff
@@ -0,0 +1,72 @@
+package org.eclipse.passage.loc.internal.workbench.wizards;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.emf.edit.domain.IEditingDomainProvider;
+import org.eclipse.passage.lic.base.LicensingResults;
+import org.eclipse.passage.lic.emf.ecore.EditingDomainRegistry;
+import org.eclipse.passage.lic.emf.edit.ClassifierInitializer;
+import org.eclipse.passage.lic.jface.dialogs.LicensingResultDialogs;
+import org.eclipse.passage.loc.internal.workbench.ClassifierMetadata;
+import org.eclipse.passage.loc.internal.workbench.i18n.WorkbenchMessages;
+import org.eclipse.passage.loc.workbench.LocWokbench;
+
+/**
+ * Creates new root licensing object. Can be asked for a reference to a created
+ * instance.
+ * 
+ * @since 0.6
+ *
+ */
+public final class RootClassifierWizard extends BaseClassifierWizard<RootClassifierWizardPage> {
+
+	private final EditingDomainRegistry<?> domainRegistry;
```

Comment:
It looks like we can avoid [compound name](https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html) here. How do you think?